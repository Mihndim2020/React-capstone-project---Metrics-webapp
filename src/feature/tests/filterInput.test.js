import React from 'react';
import renderer from 'react-test-renderer';
import { Provider } from 'react-redux';
import { BrowserRouter as Router } from 'react-router-dom';
import Filter from '../components/pages/filterInput';
import store from '../redux/slices/configStore';

it('renders correctly', () => {
  const tree = renderer
    .create(<Router><Provider store={store}><Filter /></Provider></Router>)
    .toJSON();
  expect(tree).toMatchSnapshot();
});
