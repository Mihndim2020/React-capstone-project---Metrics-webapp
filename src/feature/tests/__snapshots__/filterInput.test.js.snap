// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="deaths-filter"
>
   Filter by minimum number of Deaths: 
  <input
    id="deaths-range"
    max="9999999999"
    min="0"
    name="death-range"
    type="number"
  />
</div>
`;
